cmd/tigerblood-cli/cmd/ban.go

```diff
@@ -35,33 +34,19 @@ var banCmd = &cobra.Command{
 			viper.GetString("HAWK_ID"),
 			viper.GetString("HAWK_SECRET"))
 		if err != nil {
-			fmt.Printf("Error creating tigerblood client:\n%s\n", err)
+			fmt.Fprintf(os.Stderr, "Error creating tigerblood client: %s\n", err)
 			os.Exit(1)
 		}
 
-		resp, err := client.BanIP(cidr)
+		_, err = client.BanIP(cidr)
 		if err != nil {
-			fmt.Printf("Error banning IP:\n%s\n", err)
-			os.Exit(1)
-		} else if resp.StatusCode == http.StatusCreated || resp.StatusCode == http.StatusOK {
-			fmt.Printf("Banned %s on %s\n", cidr, url)
-		} else {
-			fmt.Printf("Bad response banning IP:\n%s\n", resp)
+			fmt.Fprintf(os.Stderr, "Error banning IP: %s\n", err)
```

Comment:
I think it can be useful to see the error response, could we make an issue to add that back with a verbose option?

Reply:
Added #104